Apps/WebClient/src/ClientApp/src/models/immunizationTimelineEntry.ts

```diff
@@ -24,6 +24,11 @@ export default class ImmunizationTimelineEntry extends TimelineEntry {
         text = text.toUpperCase();
         return text.includes(filter.keyword.toUpperCase());
     }
+
+    public getDisplayFormat(value: string): string {
```

Comment:
Sorry, what I meant is that ImmunizationAgentViewModel has the fields already set for that. No need to do the computation to get the display.
like:
```
Constructor(model: ImmunizationAgent) {
        ...
        this.lotNumber = model.lotNumber === "" ? "N/A : model.lotNumber;
        ....
    }
```

Reply:
sounds good, I have moved them.

Reply:
function not needed anymore now

Reply:
good catch! Thanks @FuriousLlama